resources/definitions/fdmprinter.def.json

```diff
@@ -2466,6 +2466,19 @@
                             "settable_per_extruder": true
                         }
                     }
+                },
+                "switch_extruder_extra_prime_distance": 
```

Comment:
Hi @vgribinchuk, we'd suggest to use the name `switch_extruder_extra_prime_amount` and with unit `mm3` like `wipe_retraction_extra_prime_amount`. This makes those settings more consistent. https://github.com/Ultimaker/CuraEngine/pull/882 needs to be adjusted for this.

Reply:
Hi @LipuFei ! Thank you, I will have a look.

Reply:
@LipuFei , requested changes were submitted. Could you please review these pull requests again?